src/app/demo/demo.component.html

```diff
@@ -108,5 +108,38 @@ <h2>Custom labels</h2>
         </p>
       </div>
     </div>
+    <hr />
+    <div class="row">
+      <div class="col-12">
+        <h2>Custom Switch Content</h2>
+        <p class="custom-icons">
+          <ui-switch uncheckedLabel="off"
+                     checkedLabel="on"
+                     size="small">
+            <img
+              style="height:10px; width:10px; margin-left:30%;margin-top:1%;"
+              src="/assets/icons/feather/activity.svg"/>
```

Comment:
This image won't be pulled from the assets correctly when running `ng build --prod`.

Use `assets/icons/feather/activity.svg` instead.

Reply:
Personally, I would recommend adding `font-awesome` to the top level `package.json` and `src/styles.scss`